src/react/ISV/components/Modal/ISVModal.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  Banner,
+  Form,
+  Icon,
+  Link,
+  Modal,
+  spacing,
+  Stack,
+  Text,
+  Wrap,
+} from '@scality/core-ui';
+import { Box, Button } from '@scality/core-ui/dist/next';
+import { useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { CardISV } from './CardISV';
+import { ISVList } from '../ISVList';
+import { useBasenameRelativeNavigate } from '@scality/module-federation';
+import { ArtescaLogo } from '../ArtescaLogo';
+import { ISVCardConfig } from '../../types';
+
+const CustomModal = styled(Modal)`
+  background-color: ${(props) => props.theme.backgroundLevel1};
+  > div {
+    max-width: 60vw;
+    width: 60vw;
+  }
+`;
+
+export const StyledGrid = styled.div`
+  display: grid;
+  gap: ${spacing.r8};
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-auto-rows: 7rem;
+  border-radius: ${spacing.f8};
+`;
+
+const StyledForm = styled(Form)`
+  background-color: ${(props) => props.theme.backgroundLevel2};
+  border-radius: ${spacing.f8};
+  overflow-y: auto;
+  overflow-x: hidden;
+  height: 50vh;
+`;
+
+const ModalFooter = styled(Box)`
+  display: flex;
+  width: 100%;
+  height: 64px;
+  align-items: center;
+`;
+
+export const ISVModalContent = ({
+  selectedISV,
+  setSelectedISV,
+}: {
+  selectedISV: ISVCardConfig;
+  setSelectedISV: (value: ISVCardConfig) => void;
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Stack direction="vertical" gap="r16">
+      <Text isEmphazed variant="Large">
+        Which application would you like to configure with your ARTESCA?
```

Comment:
`ARTESCA` shouldn't be hard coded anywhere in Zenko, Zenko can be used in other contexts. Maybe we can merge a first version as is and think about this one in next PR